source/_integrations/risco.markdown

```diff
@@ -27,6 +27,17 @@ An Alarm Control Panel entity will be created for each partition in your site, a
 
 If you have multiple sites, only the first site will be used.
 
+### Extra configuration of the integration
+
+You can configure additional options through the integration options flow:
+
+| Option | Description |
+| -------| ----------- |
+| How often to poll Risco (in seconds) | The lower this is, the faster your entities will reflect changes, but the more resource intensive it'll be. |
+| Require pin code to arm | When checked, you'll need to enter your pin code when arming through Home Assistant. |
+| Require pin code to disarm | When checked, you'll need to enter your pin code when disarming through Home Assistant. |
```

Comment:
We have helpers for this now. Example:

```
{% configuration_basic %}
The name of the option:
  description: "Description blablabal"
Other option:
  description: Lorem ipsum
{% endconfiguration_basic %}
```

Reply:
Thanks. Like this?

Reply:
Yep!

![image](https://user-images.githubusercontent.com/195327/91012355-3d2fc100-e5e6-11ea-8f98-134bc4808e16.png)